src/Duo.Clients.Wpf/Messaging/Handlers/ApriGestioneAnagraficaTrattamentiMessageHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Duo.Clients.Wpf.Presentation;
+using System.Windows;
+using Topics.Radical.ComponentModel.Messaging;
+using Topics.Radical.Messaging;
+using Topics.Radical.Windows.Presentation.ComponentModel;
+
+namespace Duo.Clients.Wpf.Messaging.Handlers
+{
+    class ApriGestioneAnagraficaTrattamentiMessageHandler : AbstractMessageHandler<ApriGestioneAnagraficaTrattamentiMessage>, INeedSafeSubscription
+    {
+        readonly IViewResolver viewResolver;
+        readonly IConventionsHandler conventions;
+
+        public ApriGestioneAnagraficaTrattamentiMessageHandler(IViewResolver viewResolver, IConventionsHandler conventions)
+        {
+            this.viewResolver = viewResolver;
+            this.conventions = conventions;
+        }
+        public override void Handle(object sender, ApriGestioneAnagraficaTrattamentiMessage message)
+        {
+            var view = viewResolver.GetView<GestioneAnagraficaTrattamentiView, GestioneAnagraficaTrattamentiViewModel>(vm => { });
```

Comment:
Ora da qualche parte la uso. Dalle altre parti potrei toglierla in effetti, quando sono sicuro che non mi serve "passare parametri"